subsys/bluetooth/controller/Kconfig

```diff
@@ -44,12 +44,17 @@ config BT_LL_SOFTDEVICE
 	select BT_CTLR_LE_PATH_LOSS_MONITORING_SUPPORT
 	select BT_CTLR_SUBRATING_SUPPORT
 	select BT_LL_SOFTDEVICE_HEADERS_INCLUDE
+	imply BT_CONN_TX_NOTIFY_WQ if SOC_FLASH_NRF_RADIO_SYNC_MPSL
```

Comment:
I don't think it is a good idea to add an imply here for the SoftDevice Controller. We should rather enable it by default. That being said, we shouldn't enable experimental symbols for default builds

Reply:
The symbol was marked as experimental, because it's a temporary solution and we want to eventually drop it without following deprecation policies. From my observations, enabling the option might improve stability and prevent flash operation failures.

If the solution is not accepted NCS-wide, we could enable it e.g. only for applications maintained by us. Still, looking at the possible consequences of the fixed flash issue, I think that it may be reasonable to enable it by default if MPSL flash sync is used together with Bluetooth.

Reply:
To be honest I prefer `imply` over overriding Kconfig default here. While using `imply`, we do not need to worry about Kconfig dependencies. Overriding defaults could actually result in enabling a symbol even though dependencies of the symbol are not met (we would need to maintain the same set of dependencies in place where we override default to ensure consistent behavior).

Reply:
Maybe we could introduce an artificial symbol that would be enabled by default if both SoftDevice and MPSL flash sync are enabled? The symbol could imply the workaround.